source/features/conversation-filters.tsx

```diff
@@ -6,15 +6,11 @@ import SearchQuery from '../github-helpers/search-query';
 import {getUsername} from '../github-helpers';
 
 function init(): void {
-	// Use an existing dropdown item to preserve its DOM structure (supports old GHE versions)
-	const sourceItem = select<HTMLAnchorElement>([
-		'#filters-select-menu a:nth-last-child(2)', // GHE
```

Comment:
Looks like this was wrongly labeled ⬆️ works ⬇️ does not

Reply:
Was added [Here](https://github.com/sindresorhus/refined-github/pull/2809/files)